bindings_ffi/src/mls.rs

```diff
@@ -1377,7 +1432,13 @@ mod tests {
 
         let alix_group = alix
             .conversations()
-            .create_group(vec![caro.account_address.clone()], None)
+            .create_group(
+                vec![caro.account_address.clone()],
+                FfiCreateGroupOptions {
```

Comment:
Could derive 'Default' to make all these 'FfiCreateGroupOptions::default()'.

Reply:
Learn a new rust trick every day!